dev-db/sqliteman/sqliteman-1.2.2-r4.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils flag-o-matic
+
+DESCRIPTION="Powerful GUI manager for the Sqlite3 database"
+HOMEPAGE="https://sourceforge.net/projects/sqliteman/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE=""
+
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtsql:5[sqlite]
```

Comment:
This is likely missing a `qtwidgets:5` dependency, if our cmake code ain't wrong